theme/layout/theme.liquid

```diff
@@ -1,12 +1,35 @@
 <!doctype html>
-<html>
+<html lang="en">
 <head>
   <title>{{ page_title }}</title>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <meta name="description" content="{{ page_description | escape }}">
   <link rel="canonical" href="{{ canonical_url }}">
   <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
+  <link rel="apple-touch-icon" sizes="57x57" href="{{ 'favicon.png' | asset_img_url: 57x57 }}">
+  <link rel="apple-touch-icon" sizes="60x60" href="{{ 'favicon.png' | asset_img_url: 60x60 }}">
+  <link rel="apple-touch-icon" sizes="72x72" href="{{ 'favicon.png' | asset_img_url: 72x72 }}">
+  <link rel="apple-touch-icon" sizes="76x76" href="{{ 'favicon.png' | asset_img_url: 76x76 }}">
+  <link rel="apple-touch-icon" sizes="114x114" href="{{ 'favicon.png' | asset_img_url: 114x114 }}">
+  <link rel="apple-touch-icon" sizes="120x120" href="{{ 'favicon.png' | asset_img_url: 120x120 }}">
+  <link rel="apple-touch-icon" sizes="144x144" href="{{ 'favicon.png' | asset_img_url: 144x144 }}">
+  <link rel="apple-touch-icon" sizes="152x152" href="{{ 'favicon.png' | asset_img_url: 152x152 }}">
+  <link rel="apple-touch-icon" sizes="180x180" href="{{ 'favicon.png' | asset_img_url: 180x180 }}">
+  <link rel="icon" type="image/png" sizes="192x192" href="{{ 'favicon.png' | asset_img_url: 192x192 }}">
+  <link rel="icon" type="image/png" sizes="32x32" href="{{ 'favicon.png' | asset_img_url: 32x32 }}">
+  <link rel="icon" type="image/png" sizes="96x96" href="{{ 'favicon.png' | asset_img_url: 96x96 }}">
+  <link rel="icon" type="image/png" sizes="16x16" href="{{ 'favicon.png' | asset_img_url: 16x16 }}">
+  <!-- Facebook -->
```

Comment:
I feel like, we'll want the metadata to be page specific, kind of like how we set things up in a React app using React-Helmet.

This article talks about a few shopify-specific approaches that might be good to implement: https://www.shopify.com/partners/blog/open-graph-tags